antarest/study/service.py

```diff
@@ -1962,3 +1981,15 @@ def update_matrix(
         assert_permission(params.user, study, StudyPermissionType.WRITE)
         self._assert_study_unarchived(study)
         self.matrix_manager.update_matrix(study, path, slices, operation)
+
+    def check_and_update_all_study_versions_in_database(
```

Comment:
create also an utility method (to be removed after) to update database from patch.json